src/main/java/bio/terra/tanagra/utils/FileUtils.java

```diff
@@ -42,7 +42,7 @@ public static InputStream getResourceFileStream(Path resourceFilePath) {
    */
   public static InputStream getFileStream(Path filePath) {
     try {
-      return Files.newInputStream(Path.of(filePath.toAbsolutePath().toString()));
```

Comment:
Do you know why they re-built the path again? This seems like magic for some purpose, I just can't fathom why

Reply:
I have no idea, although from looking at other code here it looks like they're not familiar with `Path` APIs in Java, so maybe there was some uncertainty there.

If we keep this code in MVP, the first thing is to rewrite it so it no longer has static state.